include/z64audio.h

```diff
@@ -195,52 +231,52 @@ typedef struct {
 
 typedef struct {
     /* 0x00 */ s16 numSamplesAfterDownsampling; // never read
-    /* 0x02 */ s16 chunkLen; // never read
+    /* 0x02 */ s16 numSamples; // never read
     /* 0x04 */ s16* toDownsampleLeft;
     /* 0x08 */ s16* toDownsampleRight; // data pointed to by left and right are adjacent in memory
     /* 0x0C */ s32 startPos; // start pos in ring buffer
-    /* 0x10 */ s16 lengthA; // first length in ring buffer (from startPos, at most until end)
-    /* 0x12 */ s16 lengthB; // second length in ring buffer (from pos 0)
-    /* 0x14 */ u16 unk_14;
-    /* 0x16 */ u16 unk_16;
-    /* 0x18 */ u16 unk_18;
-} ReverbRingBufferItem; // size = 0x1C
+    /* 0x10 */ s16 size; // first length in ring buffer (from startPos, at most until end)
+    /* 0x12 */ s16 wrappedSize; // second length in ring buffer (from pos 0)
+    /* 0x14 */ u16 loadResamplePitch;
+    /* 0x16 */ u16 saveResamplePitch;
+    /* 0x18 */ u16 saveResampleNumSamples;
+} ReverbBufferEntry; // size = 0x1C
```

Comment:
This probably should retain some overall documentation that says it's a ring buffer?

Reply:
Were you thinking via comments or the structure/names themselves? There's a few buffers that are ring buffers throughout (quite a few command buffers, this reverb sample buffer), should they all have "ring buffer" in their name, or left to comments? I'm actually not sure, I can always gather together all the ring-buffer structs in audio to get a bigger picture too

Reply:
I was only thinking in terms of a doxygen comment at the top of the struct to mention it, if there's a lot it can wait for a later pass, but wanted to mention it now since it's quite an important property of this struct, right?